.travis/build_package.sh

```diff
@@ -39,6 +33,11 @@ function build_demo {
     EXTRA_CXX_FLAGS="-Werror=inconsistent-missing-override"
   fi
   if [ $NEED_3D = 1 ]; then
+    if [ "$TRAVIS_OS_NAME" = "osx" ]; then
+  	  cmake -DCGAL_DIR="$ROOT/build" -DQGLVIEWER_INCLUDE_DIR="$ROOT/$DEMO/qglviewer/QGLViewer/QGLViewer.framework/Headers" -DQGLVIEWER_LIBRARIES="$ROOT/$DEMO/qglviewer/QGLViewer/QGLViewer.framework" -DCMAKE_CXX_FLAGS_RELEASE="-DCGAL_NDEBUG" ..
+    else
+   	  cmake -DCGAL_DIR="$ROOT/build" -DQGLVIEWER_INCLUDE_DIR="$ROOT/$DEMO/qglviewer" -DQGLVIEWER_LIBRARIES="$ROOT/$DEMO/qglviewer/QGLViewer/libQGLViewer.so" -DCMAKE_CXX_FLAGS_RELEASE="-DCGAL_NDEBUG" ..
+    fi
```

Comment:
There is probably a problem in our `FindQGLViewer` cmake script (no framework-aware), that explains why we need to tweak `DYLD_FRAMEWORK_PATH`.